### ISSUE TITLE
Typescript: CSSModules missing default function

### ISSUE THREAD

**Issue by darajava** [gajus/react-css-modules#270]
I am running the following code in Typescript. I have css-loader working properly as the styles object is populated. However, something goes wrong in the final line, where `CSSModules` is called. 

Webpack:
```js
{
    test: /\.css$/,
    use: ExtractTextPlugin.extract({
        fallback: 'style-loader',
        use: 'css-loader?modules,localIdentName="[name]-[local]-[hash:base64:6]"'
    }),
},
```

React (tsx):
```js
import * as React from "react";
import CSSModules from "react-css-modules";

const styles = require('./styles.css');

const Test = (properties: any) => {
    return (
        <div>
            <div styleName='red'>{'test'}</div>
        </div>
    );

}

export default CSSModules(Test, styles);  // <-- ERROR HERE
```
Error:

    react_css_modules_1.default is not a function

Is this an issue? Any idea what's happening here? Might this be a Typescript problem?

**Comment by darajava** [comment]
@gajus I don't think this is a question, I think it's an issue. I am still faced with the same problem.

**Comment by redaxmedia** [comment]
I have this issue too, but I cannot solve using className over styleName...

**Comment by redaxmedia** [comment]
Using `babel-plugin-react-css-modules` instead worked for me using Typescript and Jest. I suggest to use Enzyme also to get rid of the styleName warnings.

**Comment by San4oPanso** [comment]
Adding ` import * as CSSModules from 'react-css-modules'; ` fixed the issue for me. But in general would be good if someone add default import for the module.

Regarding `styleName` issue - i've added `react-extension.d.ts` file with content and included it into global typings by reference in index.d.ts  `/// <reference path="react-extension.d.ts" />`
```
import { DOMAttributes } from 'react';

declare module 'react' {
  interface HTMLAttributes<T> extends DOMAttributes<T> {
    styleName?: string
  }
}
```
